packages/terra-date-picker/src/DateInput.jsx

```diff
@@ -30,6 +30,14 @@ import styles from './DatePicker.module.scss';
 const cx = classNames.bind(styles);
 
 const propTypes = {
+  /**
+   * An identifier used by assistive technologies like screen readers to briefly describe this Date input to users.
+   * The label is not rendered visually.
+   *
+   * **BEST PRACTICE FOR ACCESSIBILITY**: you _SHOULD_ set this to match whatever visible label you give in your UI.
+   * Currently this is optional for passivity reasons, but it will become mandatory in a future major version.
+   */
+  a11yLabel: PropTypes.string,
```

Comment:
Can we make use of the existing ariaLabel prop here?

Reply:
done https://github.com/cerner/terra-framework/pull/2031/commits/8d4603715fe10fee9e460c580b7f7c9635d09565